src/containers/dashboard/purchases.svelte

```diff
@@ -2,11 +2,22 @@
   import Card from 'ui/card.svelte';
   import Button from 'ui/button.svelte';
   import PurchaseCard from '@/components/dashboard/purchase-card.svelte';
+  import { patch } from '@/utils/api.js';
+  import StockChangeStatuses from '@/constants/backend/stock-change-statuses.js';
 
   export let purchases = [];
-  // In real-case situation would be a bit different
-  const reject = e => purchases = purchases.filter(x => x.name !== e.detail.name);
-  const deliver = e => reject(e);
+  async function setStatus(stock_change, status) {
```

Comment:
it's rather strange that you went with `snake_case` for the `stock_change` variable. Not like it's coming from the backend or anything

Reply:
Was just consulting the backend for the payload and so on, so it was an unconscious choice.